README.md

```diff
@@ -40,21 +40,28 @@ moment. With good `binding.pry` skills, you will rock in the developer world.
 
 P.S: Aaron Patterson, one of the most famous Ruby developer, is a "puts
 debugger". It is another way to deal with errors. Enjoy
-[this article](https://tenderlovemaking.com/2016/02/05/i-am-a-puts-debuggerer.html)
-to learn more.
+[this article] to learn more.
 
 ### Learn to Google a problem
 
 When you are facing a problem you really don't know, get used to Google it.
 99.9% of the times, you will find somebody who had a pretty similar problem on
 StackOverflow. Google skills are the most important ones for a developer.
 
+Other places you can find some help are: [Reddit]
+and also directly on Github, in the [issues] section. There, you can ask your
+question to people working with the same tool as you. Sometimes you also get a
+reply directly from the creator of the project.
+
+I'm talking about these two because they are not as much referenced as
+StackOverflow but are sometimes more qualitative/recent !
+
 ## How to progress after Le Wagon
 
 Here are my personal advice on how to progress after Le Wagon. I myself did/read
-half of the resources and heard really good reviews from Le Wagon alumnus about
+half of the resources and heard really good reviews from Le Wagon alumnis about
```

Comment:
Actually, alumni seems to already be the plural of alumnus

http://www.dictionary.com/browse/alumni

Reply:
Yeah indeed ! xD
But I guess everybody is doing the mistake to append an `s` to alumni. x)